apps/src/code-studio/stageLockRedux.js

```diff
@@ -146,33 +139,25 @@ export const setViewType = viewType => {
   };
 };
 
-export const setSections = sections => ({
-  type: SET_SECTIONS,
-  sections
-});
-
-export const selectSection = sectionId => ({
-  type: SELECT_SECTION,
-  sectionId
-});
-
-export const openLockDialog = stageId => ({
+export const openLockDialog = (sectionId, stageId) => ({
```

Comment:
This seems good.  I wonder if the alternative here is for the lock dialog itself (which is connected, I assume) to get anything computable in its connect methods instead, such that you could change `selectedSection` out from under the dialog and it would re-render into a good state.  Not needed and I don't know how viable, but it seems a little like we're saving some derived state to the store here when it could be computed on-the-fly.